docs/walkthrough/run-job.md

```diff
@@ -73,4 +77,31 @@ run = finetuner.fit(
 
 ```{Important}
 Please check the [developer reference](../../api/finetuner/#finetuner.fit) to get the available options for `loss`, `miner`, `optimizer` and `scheduler_step`.
-```
\ No newline at end of file
+```
+
+### Configuration of the Optimizer
+Fintuner allows one to choose any of the optimizers provided by PyTorch.
+By default, the `Adam` optimizer is selected.
+To select a different one, you can specify its name in the `optimizer` attribute of the fit function.
+Possible values are: `Adadelta`, `Adagrad`, `Adam`, `AdamW`, `SparseAdam`, `Adamax`, `ASGD`, `LBFGS`, `NAdam`, `RAdam`, `RMSprop`, `Rprop`, and `SGD`.
+
+Finetuner configures the learning rate of the optimizer by using the value of the `lr` option.
+If you want to pass more parameters to the optimizer, you can specify them via `optimizer_options`.
+For example, you can enable the weight decay of the Adam optimizer to penalize high weights in the model by setting `optimizer_options={'weight_decay':0.01}`.
+
+For detailed documentation of the optimizers and their parameters, please take a look at the [PyTorch documentation](https://pytorch.org/docs/stable/optim.html).
+
+### Configuration of the Miner
+
+To filter the instances in a batch that are used to calculate the loss, you can use miners.
+Finetuner allows you to use miners provided by the [Pytorch Metric Learning](https://kevinmusgrave.github.io/pytorch-metric-learning) framework. 
+To select a specific miner, you can pass its name to the fit function, e.g., `AngularMiner`, `TripletMarginMiner`, ...
```

Comment:
you can list all miners to the developer reference field, all documented in docstring

Reply:
First I had this, but it looked ugly, because the names are so long and therefore, the list is very long.